data-pipeline/assessment-aggregator/src/main/config/assessment-aggregator.properties

```diff
@@ -69,6 +69,8 @@ job.coordinator.replication.factor=1
 
 output.failed.topic.name=__env__.telemetry.assess.failed
 output.metrics.topic.name=__env__.pipeline_metrics
+output.prometheus.metrics.topic.name=__env__.telemetry.metrics
+pipeline.metrics.list=db-hit-count,db-insert-count,db-update-count
```

Comment:
I am assuming this has been added to identify the counters to be added to the json. Is this needed? There will be a tendency that the developers might miss updating this config on addition of new metrics. Is there a simpler approach for this?

Reply:
Yes. Currently, we set all metrics in the pipeline_metrics json and most of the metrics will be 0 as not all metrics are applicable to each job. We can define a default set of metrics to fall back to if it is not set in the job.